Sources/Orbit/Components/SegmentedSwitch.swift

```diff
@@ -19,42 +21,54 @@ public struct SegmentedSwitch<Selection: Hashable, Content: View>: View {
     public var body: some View {
         FieldWrapper(label, message: message) {
             InputContent(message: message) {
-                content
-                    .environment(\.segmentSelection, selectionBinding)
-                    .overlay(separator)
-                    .backgroundPreferenceValue(IDPreferenceKey.self) { preference in
-                        selectionBackground(preference)
-                    }
-                    .fixedSize(horizontal: false, vertical: true)
+                HStack(spacing: 0) {
+                    content
+                        .allowsHitTesting(false)
+                        .frame(maxWidth: .infinity)
```

Comment:
We could add a support for `idealSize` here, as we support it in other components where it makes sense. Here, call site might want to have the switch that only takes a necessary width.